Library/ViewModels/ProjectPageViewModel.swift

```diff
@@ -99,6 +102,9 @@ public protocol ProjectPageViewModelOutputs {
   /// Emits a `Project` when the updates are to be rendered.
   var goToUpdates: Signal<Project, Never> { get }
 
+  /// Emits a `Project` when the report project view is to be rendered.
+  var goToReportProject: Signal<Project, Never> { get }
```

Comment:
Need to pass a Project here because the submission form will need to include the project url. https://kickstarter.slack.com/files/U0558SFCTL5/F05SJHQP23A/screenshot_20230911-125644.png